spec/index.bs

```diff
@@ -1255,6 +1255,34 @@ For more information on issuing CORS requests for image and video elements, cons
 - [[html#the-img-element]] <{img}>
 - [[html#media-elements]] {{HTMLMediaElement}}
 
+## Color Spaces ## {#color-spaces}
+
+WebGPU does not provide color management. All values within WebGPU (such as texture elements)
+are raw numeric values, not color-managed color values.
+
+WebGPU *does* interface with color-managed outputs (via {{GPUCanvasContext}}) and inputs
+(via {{GPUQueue/copyExternalImageToTexture()}}).
+Color conversion must be performed between the WebGPU numeric values and the external color values.
+These interface points locally define a color space, in which the WebGPU numeric values are to be
+interpreted, from options defined in [[css-color-4#predefined]].
+
+<script type=idl>
+enum GPUPredefinedColorSpace {
+    "srgb",
+};
+</script>
+
+Issue: Replace this with the `PredefinedColorSpaceEnum` from the
+[canvas color space proposal](https://github.com/WICG/canvas-color-space/blob/main/CanvasColorSpaceProposal.md).
```

Comment:
can we turn it into a real external spec link?

Reply:
This is just an issue and there's no spec yet. We could add an entry to the metadata at the top of the file so we could call it `[[CanvasColorSpaceProposal]]` or something, but ultimately this will point at the HTML spec.